photon-lib/src/main/java/org/photonvision/PhotonUtils.java

```diff
@@ -170,4 +167,40 @@ public static Pose2d estimateFieldToCamera(Transform2d cameraToTarget, Pose2d fi
         var targetToCamera = cameraToTarget.inverse();
         return fieldToTarget.transformBy(targetToCamera);
     }
+
+    /**
+     * Estimates the pose of the robot in the field coordinate system, given the pose of the fiducial tag, the robot relative to the
+     * camera, and the target relative to the camera.
+     *
+     * @param fieldRelativeTagPose Pose3D the field relative pose of the target
+     * @param cameraToRobot Transform3D of the robot relative to the camera. Origin of the robot is defined as the center.
+     * @param cameraToTarget Transform3D of the target relative to the camera, returned by PhotonVision
+     * @return Transform3d Robot position relative to the field
+     */
+    public static  Pose3d estimateFieldToRobotAprilTag(Transform3d cameraToTarget, Pose3d fieldRelativeTagPose, Transform3d cameraToRobot) {
+        return fieldRelativeTagPose.plus(cameraToTarget.inverse()).plus(cameraToRobot.inverse());
```

Comment:
Not sure this is right? we want fieldToTarget + targetToCamera + cameraToRobot, so we don't actually need the last inverse right?

Reply:
Yes, otherwise this is identical to the `ComputerVisionUtil.robotPoseFromCV` method which is in 2023 wpilib. Is this necessary in that case?

Reply:
I'd think we should just have it in ours. Next year we should consider ditching PhotonUtils and potentially look into making all of PhotonLib into WPI